Sources/muterCore/Configuration/MutationPosition.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftSyntax
+
+struct MutationPosition: Codable, Equatable {
+    let utf8Offset: Int
+    let line: Int
+    let column: Int
+
+    init(utf8Offset: Int, line: Int? = nil, column: Int? = nil) {
+        self.utf8Offset = utf8Offset
+        self.line = line ?? 0
+        self.column = column ?? 0
+    }
```

Comment:
```suggestion
    init(utf8Offset: Int, line: Int = 0, column: Int = 0) {
        self.utf8Offset = utf8Offset
        self.line = line
        self.column = column
    }
```

This way you let clear what is happening without having to look at the implementation.